src/components/SettlementButton.tsx

```diff
@@ -227,12 +213,12 @@ function SettlementButton({
             anchorAlignment={kycWallAnchorAlignment}
             shouldShowPersonalBankAccountOption={shouldShowPersonalBankAccountOption}
         >
-            {(triggerKYCFlow, buttonRef) => (
+            {(triggerKYCFlow: TriggerKYCFlow, buttonRef: ForwardedRef<HTMLElement>) => (
```

Comment:
Why not infer the types here?
```suggestion
            {(triggerKYCFlow, buttonRef) => (
```